src/Makevars.win

```diff
@@ -0,0 +1,2 @@
+CXX_STD = CXX11
```

Comment:
BTW, This is not necessary, right? Because it's already inserted via `SystemRequirements` in `DESC`

Reply:
I actually don't know...